spec.html

```diff
@@ -37430,12 +37156,29 @@ <h1>Atomics.compareExchange ( _typedArray_, _index_, _expectedValue_, _replaceme
         1. If IsDetachedBuffer(_buffer_) is *true*, throw a *TypeError* exception.
         1. NOTE: The above check is not redundant with the check in ValidateIntegerTypedArray because the call to ToBigInt or ToIntegerOrInfinity on the preceding lines can have arbitrary side effects, which could cause the buffer to become detached.
         1. Let _elementType_ be the Element Type value in <emu-xref href="#table-the-typedarray-constructors"></emu-xref> for _arrayTypeName_.
+        1. Let _elementSize_ be the Element Size value specified in <emu-xref href="#table-the-typedarray-constructors"></emu-xref> for Element Type _elementType_.
```

Comment:
This change is from https://github.com/tc39/ecma262/pull/2250; it shouldn't be in this PR. (I think this was probably a rebase issue?)

Reply:
resolved. (?)